Src/WitsmlExplorer.Frontend/components/Sidebar/MudLogItem.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useCallback, useContext, useEffect, useState } from "react";
+import NavigationContext from "../../contexts/navigationContext";
+import NavigationType from "../../contexts/navigationType";
+import GeologyInterval from "../../models/geologyInterval";
+import MudLog from "../../models/mudLog";
+import { calculateObjectNodeId } from "../../models/objectOnWellbore";
+import { ObjectType } from "../../models/objectType";
+import Well from "../../models/well";
+import Wellbore, { getGeologyIntervalLength } from "../../models/wellbore";
+import GeologyIntervalItem from "./GeologyIntervalItem";
+import TreeItem from "./TreeItem";
+import { WellboreItemContext } from "./WellboreItem";
+import { ComponentType } from "../../models/componentType";
+import ComponentService from "../../services/componentService";
+import ModificationType from "../../contexts/modificationType";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+interface ObjectOnWellboreItemProps {
+  mudlogData: MudLog;
+  objectType: ObjectType;
+}
+
+const MudLogItem = (props: ObjectOnWellboreItemProps): React.ReactElement => {
```

Comment:
I couldn't find where to trigger the context menu. The context menu for a mud log in the sidebar differs from the list view. They should be the same.